app/src/forms/teammanagement/events.js

```diff
@@ -0,0 +1,3 @@
+module.exports = Object.freeze({
+  ACCESS_REQUESTED: 'onAccessRequest'
+});
```

Comment:
Perhaps this would be better residing in the existing /utils/constants.js file?

Reply:
makes it harder to isolate if we did have to copy out a form silo.  really don't want things super dependent when there is no need.